packages/UI/src/components/Dropdown2/Dropdown.tsx

```diff
@@ -57,23 +69,16 @@ const Dropdown = ({
       </Toggle>
       <OptionsContainer isOpen={menuOpen}>
         <Options>
-          {options.map((option) => (
-            <Option key={option} htmlFor={String(option)}>
```

Comment:
Shouldn't `Option` be a `select` tag instead of `label` tag

Reply:
Do you mean `Options` instead of `Option`? Because `Option ` is for each individual option in the dropdown.